Sources/SwiftDriver/Driver/Driver.swift

```diff
@@ -13,6 +13,7 @@ import TSCBasic
 import TSCUtility
 import Foundation
 import SwiftOptions
+import _CSwiftDriver
```

Comment:
```suggestion
@_implementationOnly import _CSwiftDriver
```
Because we do not wish clients of `SwiftDriver` to have to go looking for this module, I think it should be used with `@_implementationOnly`. Otherwise even `SwiftDriverExecution` target will not build with CMake because it will go looking for the modulemap of this new C target and not know where to find it.

Reply:
That's reasonable. Fixed.